db/migrate/20230103220258_create_transactions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTransactions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :transactions do |t|
+      t.references :invoice, foreign_key: true
+      t.integer :credit_card_number, limit: 8
```

Comment:
Very interesting find on how integers have byte limits

Reply:
Yes! The 'limit' symbol triggers the bigint column type in Postgres.